lib/handlers/personalaccesstoken.ts

```diff
@@ -18,10 +18,11 @@ export class PersonalAccessTokenCredentialHandler implements ifm.IRequestHandler
     }
 
     // This handler cannot handle 401
-    canHandleAuthentication(res: ifm.IHttpResponse): boolean {
+    canHandleAuthentication(response: ifm.IHttpClientResponse): boolean {
         return false;
     }
 
-    handleAuthentication(httpClient, protocol, options, objs, finalCallback): void {
+    handleAuthentication(httpClient: ifm.IHttpClient, requestInfo: ifm.IRequestInfo, objs): Promise<ifm.IHttpClientResponse> {
```

Comment:
lots of interface changes. is this because it's still in preview?

Reply:
Yes. This is a symptom of this code having been copied from another repo then modified. Also for these handlers since NTLM is the only one that can handleAuthentication and it wasn't being used before I don't think we break anything with this specific change. PAT uses prepareRequest which didn't change signature.